### ISSUE TITLE
User Story: Show all accounts

### ISSUE THREAD

**Issue by danielsomerfield** [somesyd/single-budget#5]
As a holder of separate USD and CAD Budgets, I would like to see all my accounts in the CAD Budget whether the source account is in CAD or USD. All accounts should be in a common CAD currency based on a static conversion rate.

# Details
I would like to go to https://app.ynab.com/_<cad-budget-id\>_ where _<cad-budget-id\>_ is the id of the CAD budget and _<usd-budget-id\>_ is the id of the USD budget and see a complete list of under "All Accounts".

# Acceptance Criteria
* For each USD account in the account at https://app.ynab.com/_<usd-budget-id\>_, I have an account in https://app.ynab.com/_<cad-budget-id\>_ under "All Accounts". These accounts might require manual setup.
* For each transaction in the USD budget less than 24 hours old, there is an equivalent transaction in it's "peer" account in the CAD budget.
* The "All Accounts" list shows each account with a total value equivalent to a total of all the above transactions, which is, in term, equivalent to the total in the USD budget times the static conversion rate.
* Each transaction in the CAD budget has been converted using a static conversation rate from USD to CAD.

# Assumptions
* I already have an existing YNAB account.
* Within that account I have a budget that is all USD accounts and another one that is all CAD accounts.

# Out of scope
* User setup of budgets and accounts - this can be done manually by admin
* Changes to USD/CAD conversion rate

# Example
Using our current budget, you can see the CAD budget at https://app.ynab.com/bdb01318-11f1-4dc2-96ae-83c6b5c16000/budget. The budget list includes:
* `Vancity CAD Checking`
* `Vancity Visa`
* `Vancity CAD Savings #2`
* `Vancity CAD Savings #3`
* `RBC CA Chequing`
* `RBC CA Savings`
* `RBC CA CC`
<img width="322" alt="image" src="https://github.com/somesyd/single-budget/assets/6019932/09f0a72a-ddc0-4b80-ac90-65304dd39265">

It also contains some additional loan and tracking accounts. These are out of scope.

If you go to the [US budget](https://app.ynab.com/00f5769e-301b-44a9-9b14-e88a522dfc73/budget), you will see the following accounts:

* `Alaska Air Visa`
* `Vancity USD Checking"
* `RBC US Checking`
* `RBC US Money Market`
* `RBC US Visa`

It too contains some additional tracking accounts that are out of scope.

<img width="330" alt="image" src="https://github.com/somesyd/single-budget/assets/6019932/dd3a2e60-dca8-40e5-a2fb-efd6931e3ada">

When this feature is done and applied to the account, the CA account will contain all the accounts listed in US that we choose to have imported, as well as the US accounts. They would have balances adjusted based on the static conversion and be updated at least once in 24 hours.

In other words, the list in the [CAD budget](https://app.ynab.com/bdb01318-11f1-4dc2-96ae-83c6b5c16000/budget) would be

* `Vancity CAD Checking`
* `Vancity Visa`
* `Vancity CAD Savings #2`
* `Vancity CAD Savings #3`
* `RBC CA Chequing`
* `RBC CA Savings`
* `RBC CA CC` 
* _Alaska Air Visa_
* _Vancity USD Checking_
* _RBC US Checking_
* _RBC US Money Market_
* _RBC US Visa_

with the highlighted ones being new to the CAD budget. The US budget list would be unchanged. 

One note: We might not want to name the the same as the above. We might want a convention like "Imported: _account name_". That decision isn't critical for the story, so whatever you choose to do is fine.
